src/main/java/com/devcycle/sdk/server/local/api/DVCLocalClient.java

```diff
@@ -119,38 +122,102 @@ public <T> Variable<T> variable(User user, String key, T defaultValue) {
       }
       return defaultVariable;
     }
-    String variableJSON = null;
+
+    double appBuild = Double.NaN;
+    try{
+      appBuild = Double.parseDouble(user.getAppBuild());
+    }catch(Exception e){}
+
+    DVCUser_PB user_pb = DVCUser_PB.newBuilder()
+            .setUserId( user.getUserId())
+            .setEmail(createNullableString(user.getEmail()))
+            .setName(createNullableString(user.getName()))
+            .setLanguage(createNullableString(user.getLanguage()))
+            .setCountry(createNullableString(user.getCountry()))
+            .setAppBuild(createNullableDouble(appBuild))
+            .setAppVersion(createNullableString(user.getAppVersion()))
+            .setCustomData(createNullableCustomData(user.getCustomData()))
+            .setCustomData(createNullableCustomData(user.getPrivateCustomData()))
+            .build();
```

Comment:
This `variable` method is getting pretty big. Could we put this as a Static method on DVCUser_PB where you pass in the `user` object?

Reply:
or if you can't modify the PB classes you could put it on the User class.

Reply:
i like that. I'll push it into the User.  Might look at that for .Net too since I kind of ported this over from there